blank-canvas-blocks/sass/blocks/_image.scss

```diff
@@ -0,0 +1,12 @@
+.wp-block-image {
+	/* 
+	From what I can tell the below are styles regularly used by themes
+	to fix the image block.  I believe these should go into the block's
+	default styles.  It's difficult to say how this will land, however 
+	based on discussion found in (many) related issues here: 
+	https://github.com/WordPress/gutenberg/issues/28923
+	*/
+	text-align: center;
+	margin: 0;
```

Comment:
Are these margin styles needed? Asking because I saw a recent Gutenberg PR that reset the margins for the image block: 

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/style.scss#L1-L3

Reply:
Oh!  I hope not.  That would be much better.  I'm not sure why that isn't reflecting for me though... I pulled and built gutenberg today and that should have picked up that change.